src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    static String getRandomBall() {
+        Random random = new Random();
```

Comment:
The Random object should be a private class-level field instead of being instantiated every time the method is called. This is more efficient as it avoids unnecessary object creation.

Reply:
Still not implemented